features/LAY000_Layers-information.feature

```diff
@@ -0,0 +1,19 @@
+@implementer-agreement
+@LAY
+@version1
+@E00020
+
+Feature: LAY000 - Layers Information
```

Comment:
Better english would be layer, but I would also really add the word Presentation, so that it's not to be confused with the physical material layers for compound structures such as walls and slab (IfcMaterialLayerSet). Lastly since again it doesn't really contain information *about* these layers, I would call it assignment. And then accidentally I have arrived at the verbatim entity name.

```suggestion
Feature: LAY000 - Presentation Layer Assignment
```

Reply:
[Modified it](https://github.com/buildingSMART/ifc-gherkin-rules/pull/256/commits/1487b2ecafdb2edea99b9ca5f5d4d69269222745)
We'll also have to modify the feature filename in that case.